app/webpacker/stylesheets/_notice-summary.scss

```diff
@@ -0,0 +1,38 @@
+.app-notice-summary {
```

Comment:
We can remove this and the other *-summary.scss if we don't include the flash partial.

Reply:
Removed.